src/button/validate.js

```diff
@@ -127,6 +128,29 @@ export function validateButtonStyle(style : Object = {}) {
             throw new Error(`style.${ BUTTON_STYLE_OPTIONS.TAGLINE } is not allowed for ${ BUTTON_LAYOUT.VERTICAL } layout - got ${ tagline }`);
         }
     }
+
+    validateRegionSpecificButton(style, props.locale);
```

Comment:
Shall we also add some validation to make sure installmentperiod is a number, and one of the accepted values we expect?

Reply:
yes